src/poliastro/tests/test_plotting.py

```diff
@@ -69,6 +69,6 @@ def test_color():
 
 
 def test_plot_solar_system():
-    op = plot_solar_system()
-    assert len(op._orbits) == 8
-    assert isinstance(op, OrbitPlotter)
+        assert len(plot_solar_system(True).orbits) == 8
```

Comment:
What happened with the indentation here?

Reply:
It looks fine in my editor :( Will check it